Sources/Purchasing/OfferingsManager.swift

```diff
@@ -91,6 +91,35 @@ class OfferingsManager {
         }
     }
 
+    func updateProductsEntitlementsCacheIfStale(
```

Comment:
Hmm so this is using `ProductsEntitlementsCache` vs `ProductEntitlementMappingResponse` for the response object... I think the differences in plurals vs singulars and the absence of `Mapping` could be confusing... But it's true that it would be a pretty long name... Maybe just `updateProductEntitlementMapCacheIfStale`?

Reply:
Fixed.